.github/workflows/gem-push.yml

```diff
@@ -0,0 +1,32 @@
+name: Ruby Gem
+
+on:
+#   push:
+#     branches: [ master ]
+  # run manually for now
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: Build + Publish
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Ruby 2.6
+      uses: actions/setup-ruby@v1
+      with:
+        ruby-version: 2.6.x
+
+    - name: Publish to RubyGems
+      run: |
+        mkdir -p $HOME/.gem
+        touch $HOME/.gem/credentials
+        chmod 0600 $HOME/.gem/credentials
+        printf -- "---\n:rubygems_api_key: ${GEM_HOST_API_KEY}\n" > $HOME/.gem/credentials
+        gem build *.gemspec
+        gem push *.gem
+      env:
+        GEM_HOST_API_KEY: "${{secrets.RUBYGEMS_AUTH_TOKEN}}"
```

Comment:
Nitpicking, but could we add the `env` line above the `run`? I find it slightly easier to read the `GEM_HOST_API_KEY ` being set, then how it's used.

Reply:
Fixed